main.go

```diff
@@ -103,12 +103,12 @@ var cmdInstall = cli.Command{
 	Action: func(c *cli.Context) error {
 		vers := c.Args().First()
 		if vers == "" {
-			stump.Fatal("please specify a version to install")
+			stump.Fatal("Please specify a version to install.")
```

Comment:
So, go error messages generally don't capitalize or have punctuation. That's probably why we did it this way. I'd rather be consistent with that (unless it looks really ugly).